wazo_sysconfd/plugins/systemd_services/http.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from fastapi import APIRouter, Body
+from .services import services
+
+router = APIRouter()
+
+@router.post('/systemd_services', status_code=200)
```

Comment:
Please don't break the API in this refactoring branch: there are other services depending on the `/services` API (confd, setupd, maybe others). Please keep the API spec as it was. And yes, I totally agree that a `/services` API is a terrible name.

Reply:
Reverted